tests/test_encounter/test_windows.py

```diff
@@ -39,7 +34,7 @@ def test_windows(self):
         indices = np.split(indices, np.where(np.diff(indices) != 1)[0] + 1)
```

Comment:
There seem to be indexed values here again, try and either use enums, set up an interface, or assign local meaning to the indices.